apply_extra.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+FLATPAK_ID=com.tracktion.Waveform13
+
+function extract_deb () {
+    cat $1 | bsdtar x $2
+    tar -xf $2 -C deb-package
+    rm $2 # Clean up tar archive
+}
+
```

Comment:
bsdtar -Oxf tracktion-download-manager.deb 'data.tar.xz' | bsdtar -xf -

use this